gensrc/thrift/PlanNodes.thrift

```diff
@@ -414,6 +414,7 @@ struct THashJoinNode {
 
   52: optional TJoinDistributionMode distribution_mode;
   53: optional list<Exprs.TExpr> partition_exprs
+  54: optional list<Types.TSlotId> output_columns
```

Comment:
You should add UT to check the thrift `output_columns`

Reply:
done